sponge/src/main/java/net/playeranalytics/plugin/SpongePluginInformation.java

```diff
@@ -27,6 +31,34 @@ public File getDataFolder() {
 
     @Override
     public String getVersion() {
-        return plugin.getClass().getAnnotation(Plugin.class).version();
+        // Implementation to convert a ArtifactVersion to String, regardless of implementation
+        ArtifactVersion artifactVersion = plugin.metadata().version();
```

Comment:
Does API 8 enforce a specific version schema? This might be problematic since Plan doesn't exactly match semantic versioning, using `<major>.<minor> build <build>` 🤔

Reply:
Well yes, but actually no. Atleast the default implementation will just treat such a version string as `0.0.0` builder number `0` and throw the entire thing into the qualifier, I adjusted the handling so that it won't be prefixed with `-` in such a case. Below is me testing it,

```
import org.apache.maven.artifact.versioning.ArtifactVersion
import org.apache.maven.artifact.versioning.DefaultArtifactVersion

field ArtifactVersion ver = 1.0 build 2

ver.getMajorVersion() = 0
ver.getMinorVersion() = 0
ver.getIncrementalVersion() = 0
ver.getBuildNumber() = 0
ver.getQualifier() = "1.0 build 2"
ver.toString() = "1.0 build 2"
```

Reply:
Alright